design/one-pager-default-resource-class.md

```diff
@@ -88,66 +116,237 @@ spec:
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstancePolicy
+metadata:
+  name: mysql-policy
+  namespace: crossplane-system
+defaultClassRef:
+  kind: RDSInstanceClass
+  apiVersion: database.aws.crossplane.io/v1alpha1
+  name: standard-mysql
+  namespace: crossplane-system
+```
 
-Finally, the controller will set the `ClassRef` of the claim to the discovered default class. The claim will now pass the predicates of the resource claim controller for the specified resource kind, and will be bound using the default class implementation.
+Then, for any `MySQLInstance` claim that was created in namespace
+`crossplane-system` without a `classRef`, the `MySQLInstance` default class
+controller would automatically assign the class which was referenced by the
+`MySQLInstancePolicy` in that namespace:
 
-## Future Considerations
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstance
+metadata:
+  name: mysql-claim
+  namespace: crossplane-system
+spec:
+  engineVersion: "9.6"
+```
 
-As Crossplane evolves, it is possible that the implementation of this functionality is manifested slightly differently. One area that might affect implementation is the introduction of [strongly typed resource classes](https://github.com/crossplaneio/crossplane/issues/90). However, the workflow for developers and operators would remain largely the same in regards to usage of default resource classes.
+## Proposal: Default Class Reference v2 & Claim Portability
 
-Additionally, some resources in Crossplane that are not portable (i.e. do not have comparable resources across providers) may eventually be implemented as their own concrete resource types. This may introduce the desire to have multiple resource claim kinds for which a resource class serves as default. For example, a claim may specify its `Kind` as either a `NoSQLInstance` or a more specific `DynamoDBInstance` and operators may want to specify the same default resource class for both of these claim kinds. The proposed implementation in this document would make this functionality possible because `defaultForClaimKinds` allows for multiple values.
+The `Policy` method continues to enable default class references, but
+strongly-typed resource classes introduce a reduction in portability of resource
+claims. Previously, claims could reference a generic `ResourceClass` by `name`
+and `namespace`, and could be moved across providers as long as there existed a
+class with that `name`. Now, claims must omit a `classRef` and rely on the
+existence of a `Policy` to achieve portability.
```

Comment:
Did we already define the term `portable` and its variants? I am realizing this would probably be good to define, in case any readers are not sure what we mean by it.

Reply:
I don't believe "portable" is defined anywhere in this doc, and I agree that it is important to do so.